near/src/runtime.rs

```diff
@@ -635,6 +637,8 @@ impl RuntimeAdapter for NightshadeRuntime {
             epoch_length: self.genesis_config.epoch_length,
             gas_price,
             block_timestamp,
+            // NOTE: verify transaction doesn't use gas limit
+            gas_limit: None,
```

Comment:
should we use gas limit for verification as well?
because if block limit is small-ish and tx is gigantic, it won't fit in the block as well?

Reply:
We probably shouldn't have TXs that are large enough to not fit into the block. We'll be able to refactor this later.